docs/getting_started/binaries.rst

```diff
@@ -88,16 +88,36 @@ The Linux binary builds can be run on Windows using the
 when solvers depend on POSIX standards.  Currently the optional PETSc solver is
 the only one which does not run on Windows.
 
-In the WSL environment, download and extract the desired Linux release containing
-the solver you would like to run.  In the Windows IDAES binary directory, create a
-batch file with the format ``{solver}_wsl.bat`` for example for PETSc
-``petsc_wsl.bat``.
+In the WSL environment, download and extract the desired Linux release of
+the solver (i.e. PETSc) you would like to run following the instructions under manual
+installation above. Install required libraries libfortran, libblas, liblapack i.e.
+``apt-get install liblapack3 libblas3 ligbomp1 libgfortran5``.
```

Comment:
Most people will probably use Ubuntu 20.04, so I like this to make it easy, but it's not for sure which distro they will use.  May want to change this to e.g. for Ubuntu 20.04 ...  Then maybe link the part of the Linux install instructions, which has details for supported distros. Admittedly it's a bit weird because we just want to reference a bit of the Linux install instructions and not the whole thing, but if we copy the relevant part we'll have to remember to update both sections as supported distros shift ahead.

Reply:
I have updated the documentation to include a line at the end with a link providing information on how to see the available Linux distributions for WSL, and to change the default distribution.

Reply:
The documentation has been updated to provide a link to the IDAES install instruction for the required libraries.